crates/starknet-server/src/api/json_rpc/mod.rs

```diff
@@ -128,6 +128,11 @@ impl<T: Serialize> ToRpcResponseResult for RpcResult<T> {
                     message: err.to_string().into(),
                     data: None,
                 },
+                ApiError::SerializationError(error) => RpcError {
```

Comment:
Shouldn't this kind of error be a signal of a bug in our code?
Anyway, this way we will be displaying a "Serialization error" message to users, without any explanation and with a potential debugging difficulty.